index.js

```diff
@@ -76,10 +76,15 @@ instance.prototype.init = function() {
 		});
 
 		self.obs.on('ConnectionClosed', function() {
-			self.log('error','Connection lost to OBS.');
-			self.status(self.STATUS_ERROR);
-			self.destroy();
-			self.init();
+			if (self.disable != true) {
+				self.log('error','Connection lost to OBS.');
+				self.status(self.STATUS_ERROR);
+				self.destroy();
```

Comment:
Hold the phone ... I don't think `self.destroy();` was supposed to be in here to begin with.  Only the core should be calling a module's destroy function

Reply:
Yeah this module is definitely a touch strange, with it's TCP to OBSWebSocket handoff.

I removed the references to destroy and replaced them with disconnect. Still needed a a if (disable) statement otherwise it gets caught in a loop re-init the module and doesn't properly disconnect

Reply:
Works for me.  Thx